### PR TITLE
fix(core): ldml: fix spelling of maqtugħa 🙀

### PR DESCRIPTION
**Pull request by srl295**
- maqtua should be maqtugħa (cut) but spelled as maqtugha where used as an NMTOKEN

@keymanapp-test-bot skip

-----

Minor, just an embarrassing typo…

### CODE REVIEW

**Comment by srl295** [comment]
protip: check the spelling, when you fix the spelling.

Should be ready to go now

File: developer/src/kmc-keyboard/test/fixtures/basic.txt [hunk 1]
```diff
@@ -276,11 +276,11 @@ block(key2)                         # struct COMP_KMXPLUS_KEY2 {
   00 00 00 00                       #   KMX_DWORD flickCount
   00 00 00 00  00 00 00 00  00 00 00 00 # Reserved[3]
   # keys
-  #  hmaqtua
+  #  hmaqtugha
   00 00 00 00                       #   KMX_DWORD vkey
   index(strNull,strKey1,2)          #   KMXPLUS_STR 'U+0127'
   01 00 00 00                       #   KMX_DWORD (flags: extend)
-  index(strNull,strHmaqtua,2)       #   KMXPLUS_STR 'hmaqtua'
+  index(strNull,strHmaqtugha,2)       #   KMXPLUS_STR 'hmaqtugha'
```
Comment (mcdurdin) [inline developer/src/kmc-keyboard/test/fixtures/basic.txt:283]:
```suggestion
  index(strNull,strHmaqtugha,2)     #   KMXPLUS_STR 'hmaqtugha'
```

File: developer/src/kmc-keyboard/test/fixtures/basic.txt [hunk 2]
```diff
@@ -354,7 +354,7 @@ block(layr)                         # struct COMP_KMXPLUS_LAYR {
   00 00 00 00                       #   KMX_DWORD key index
   02 00 00 00                       #   KMX_DWORD count
   # keys
-  index(strNull,strHmaqtua,2)       #   KMXPLUS_STR locale;       // 'hmaqtua'
+  index(strNull,strHmaqtugha,2)       #   KMXPLUS_STR locale;       // 'hmaqtugha'
```
Comment (mcdurdin) [inline developer/src/kmc-keyboard/test/fixtures/basic.txt:357]:
```suggestion
  index(strNull,strHmaqtugha,2)     #   KMXPLUS_STR locale;       // 'hmaqtugha'
```

**Review by mcdurdin — APPROVED** [review]
LGTM whitespace nits

**Review by jahorton — APPROVED** [review]

**Pull request merged**